molpipeline/estimators/chemprop/models.py

```diff
@@ -110,6 +173,29 @@ def _predict(
                 )
         return prediction_array
 
+    def fit(
+        self,
+        X: MoleculeDataset,
+        y: Iterable[int | float] | npt.NDArray[np.int_ | np.float_],
```

Comment:
Use rather a Sequence instead of Iterable. Sklearn should expect that the data is fully in memory.

Reply:
Done